lib/validation/ValidationError.js

```diff
@@ -49,7 +49,7 @@ class ValidationError extends Error {
 			separator += chalk.grey.dim("\u2500".repeat(process.stdout.columns || 80));
 		}
 		separator += "\n\n";
-		let message = chalk.red(`Invalid ui5.yaml configuration for project ${this.project.id}`) + "\n\n";
+		let message = chalk.red(`Invalid configuration for project ${this.project.id}`) + "\n\n";
```

Comment:
I validated this PR with https://github.com/SAP/ui5-project/pull/494 and noticed that I can't really have a project at hand when validating a workspace file (see https://github.com/SAP/ui5-project/pull/494/commits/2182c2905e836e36ee54f007759109c20dc7bfe1#diff-416284583829e166b5b22422140544273617f7ecdeda4e9cceb4f7e698890241R283).

In fact, the error message is also slightly misleading. Since we typically only refer to `ui5.yaml`-configuration as "project configuration", I would maybe expect something like `Invalid workspace configuration` (without referencing any project).

Maybe we can add an additional parameter to have a different error message for invalid workspace configurations?